### ISSUE TITLE
Angular2 compatibility

### ISSUE THREAD

**Issue by doshprompt** [doshprompt/angular-localization#71]
Make app compatible with angular2 and/or rollout new fork `ng2-localization`

**Comment by liquidboy** [comment]
Hi dosh,

Pardon the direct approach BUT has this ng2 work started?

I'm currently evaluating localization solutions for an angular2 project im working on and would love to use angular-localization if its compatible with ng2

**Comment by doshprompt** [comment]
@liquidboy Nope, unfortunately I haven't found the time to build ng2 compat but I believe there are also working on a native solution. Would love some help building this out!

**Comment by liquidboy** [comment]
Give me a bit of time to process the state of ng2's localization and also get my head around angular-localization ...  will love to contribute this if this is what we end up using :)

**Comment by doshprompt** [comment]
Cool, thanks a lot @liquidboy, let me know what you think of ng2 localization, it would be good to know from the POV of someone else :)
